package.json

```diff
@@ -16,11 +16,13 @@
     "qdequele <quentin@meilisearch.com>"
   ],
   "license": "MIT",
-  "main": "./dist/bundles/meilisearch.umd.js",
+  "main": "./dist/bundles/meilisearch.cjs.js",
   "module": "./dist/bundles/meilisearch.esm.js",
-  "browser": "./dist/bundles/meilisearch.umd.js",
+  "browser": "./dist/bundles/meilisearch.esm.js",
   "typings": "./dist/types/index.d.ts",
   "types": "./dist/types/index.d.ts",
+  "jsdelivr": "./dist/bundles/meilisearch.umd.js",
+  "unpkg": "./dist/bundles/meilisearch.umd.js",
```

Comment:
We could create just `"umd": "./dist/bundles/meilisearch.umd.js"`? And use it for both?

Reply:
These fields are conventions used by npm and other registries like `jsdelivr` and `unpkg` to know which bundle they need to use

Reply:
Oh I see that coming :/ Thanks for the answer :)